Mathlib/Topology/Category/TopCat/Limits/Basic.lean

```diff
@@ -184,7 +184,7 @@ instance forgetPreservesColimits : PreservesColimits (forget : TopCat.{u} ⥤ Ty
 /-- The terminal object of `Top` is `PUnit`. -/
 def isTerminalPUnit : IsTerminal (TopCat.of PUnit.{u + 1}) :=
   haveI : ∀ X, Unique (X ⟶ TopCat.of PUnit.{u + 1}) := fun X =>
-    ⟨⟨⟨fun _ => PUnit.unit, by continuity⟩⟩, fun f => by ext; aesop⟩
+    ⟨⟨⟨fun _ => PUnit.unit, continuous_const⟩⟩, fun f => by ext; aesop⟩
```

Comment:
I am not sure there is a need for this change. If `by continuity` works in reasonable time, there is no need to change it by an explicit term.

Reply:
This change is required because, after moving `sphere` and `disk` into `TopCat/Basic.lean`, the tactic `continuity` (presumably) gets overwhelmed by newly imported definitions, resulting in the timeout error `tactic 'aesop' failed, failed to prove the goal after exhaustive search`. I didn't look into the exact cause but was able to fix this by using the explicit term `continuous_const`.

For details, please look at [this conversation](https://github.com/leanprover-community/mathlib4/pull/12502#discussion_r1745581711).

This change is unrelated to CW-complexes but is required for this PR to build successfully. Should I open another PR for this change?